Lib/test/test_call.py

```diff
@@ -1048,9 +1048,12 @@ def get_sp():
 
         this_sp = _testinternalcapi.get_stack_pointer()
         lower_sp = _testcapi.pyobject_vectorcall(get_sp, (), ())
-        self.assertLess(lower_sp, this_sp)
+        if _testcapi._Py_STACK_GROWS_DOWN:
+            self.assertLess(lower_sp, this_sp)
+        else:
+            self.assertGreater(lower_sp, this_sp)
         # Add an (arbitrary) extra 25% for safety
-        safe_margin = (this_sp - lower_sp) * 5 / 4
+        safe_margin = abs(this_sp - lower_sp) * 5 / 4
```

Comment:
Can you set `safe_margin` in the if/else and add the 25% here (`safe_margin = safe_margin * 5 / 4`), to avoid `abs()`?